src/javascript/app_2/Modules/Trading/Components/Form/Purchase/contract_info.jsx

```diff
@@ -31,26 +31,24 @@ const ContractInfo = ({
                 </div>
                 :
                 <div className='purchase-info-wrapper'>
-                    <div className='stake-wrapper'>
-                        <span>
-                            <strong>{localize('Stake')}:</strong>
-                            <Money amount={proposal_info.stake} currency={currency} />
-                        </span>
-                        <span className='field-info'>
-                            <Tooltip alignment='left' icon='info' message={proposal_info.message}/>
-                        </span>
+                    <span className='purchase-tooltip'>
+                        <Tooltip alignment='left' icon='info' message={proposal_info.message}/>
```

Comment:
Can we simply use the `tooltip` property of `Fieldset` component?

Reply:
Sure, I'll update after 4PM. ;)

Reply:
@zakibinary Noted, with thanks for your effort.